setup.py

```diff
@@ -46,5 +46,9 @@ def requirements_from_pip(filename='requirements.txt'):
                       "all": all_deps},
       include_package_data=True,
       zip_safe=False,
-      classifiers=['Programming Language :: Python :: 3.6'])
-
+      classifiers=[
+          'Programming Language :: Python :: 3.6',
+          'Programming Language :: Python :: 3.7',
+          'Programming Language :: Python :: 3.8',
+          'Programming Language :: Python :: 3.9'
+          ])
```

Comment:
Shouldn't `python_requires` be also changed to `python_requires='>=3.6.2,<3.10'`?